db/migrate/20170613153960_change_numberic_value_to_double.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeNumbericValueToDouble < ActiveRecord::Migration
+  def up
+    change_column :custom_field_values, :numeric_value, :double
```

Comment:
How about using `DECIMAL` instead? Upgrading to `double` feels a bit like just pushing the border a bit, but we still don't handle exact numeric representation. With decimals we'd still need to set limit, but at least representation will be exact.